src/Application/test/LimitCalculatorGTest.cpp

```diff
@@ -0,0 +1,52 @@
+///////////////////////////////////////////////////
+/**
+ * @file LimitCalculatorGTest.cpp
+ * @brief Unit tests for checking limits with calculator
+ */
+///////////////////////////////////////////////////
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+#include<limits>
+
+#include "CalculatorApplication.h"
+#include "CalculatorApplicationFactory.h"
+#include "CalculatorStrings.h"
+
+using namespace calculator;
+using namespace testing;
+
+class whenTestingLimitCalculator : public :: testing :: Test
```

Comment:
If you aren't using the fixture... just remove this class and then change all of your `TEST_F(...)` to `TEST(...)`